README.md

```diff
@@ -2,4 +2,114 @@
 
 ## Marine Rust SDK
```

Comment:
# Marine Rust SDK
(since this is the main header). Also, please consider moving the reference to crates below the title as it is done in https://github.com/fluencelabs/marine and some other READMEs.